pages/conduct/faq.md

```diff
@@ -19,21 +19,17 @@ However, we do expect that people will abide by the spirit and words of the CoC
 
 In practice, this means the various jQuery Foundation IRC channels (#jquery, #jquery-dev, etc.), bug tracking and code review tools, and "official" jQuery Foundation events such as sprints. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
-### What about events funded by the jQuery Foundation?
+### What about jQuery Foundation events?
 
-This Code of Conduct also covers any events that the jQuery Foundation funds. However, events funded by the jQuery Foundation already [require a code of conduct](https://jquery.org/conduct/). Isn't this redundant?
+This Code of Conduct also covers all [jQuery Foundation events](http://events.jquery.org/) since it is all about how we interact as a community. It's about saying that the jQuery Foundation community will be open, friendly, and welcoming. The core issue is about ensuring the conversations we have are productive and inviting for all.
```

Comment:
I'd rather just wait. It's consistent with all the other links and the switch to HTTPS isn't really related to the content.